src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -117,6 +144,45 @@ object VersionInjectorPlugin extends AutoPlugin {
         gitConfFile
     }
 
+  val injectCacheKeyTask =
+    injectCacheKey <<= (resourceManaged in Compile, organization, name, version, streams, cacheKey) map {
+      // scalastyle: ignore
+      (resourceManaged, org, name, version, s, cacheKeyResult) =>
+        val cacheKeyConfFile = resourceManaged / org / cleanArtifactName(name) / "cacheKey.conf"
+        val cacheKeyContents = "cacheKey: " + "\"" + cacheKeyResult + "\""
+        IO.write(cacheKeyConfFile, cacheKeyContents)
+        cacheKeyConfFile
+    }
+
+  lazy val gitMRCs = Def.taskDyn {
```

Comment:
`gitMRC` should have a type declaration - it's not clear from reading what it is.